elliott/elliottlib/runtime.py

```diff
@@ -384,20 +384,6 @@ def resolve_shipment_metadata(self, strict=True):
             if '@' in self.shipment_path:
                 shipment_path, commitish = self.shipment_path.split('@')
 
-            # TODO: find a better solution
```

Comment:
So we don't need the `GITLAB_TOKEN` anymore?

Reply:
That's right it is not really needed here. It is needed when the repo is private / or we want to make pushes to it, both aren't applicable here.

Reply:
Not sure why my comment never got posted .. 
but yes it is not needed - it is needed when the repo is private or we want to push to it - neither of those is applicable here